pyface/__init__.py

```diff
@@ -22,7 +22,7 @@
 __extras_require__ = {
     "wx": ["wxpython>=4", "numpy"],
     "pyqt": ["pyqt>=4.10", "pygments"],
-    "pyqt5": ["pyqt5", "pygments"],
+    "pyqt5": ["pyqt5", "PyQtWebEngine", "pygments"],
```

Comment:
This is only needed for `pyqt5` >= 5.12, so probably should have that encoded in the PyQt version.

Reply:
Did you mean `"pyqt5": ["pyqt5>=5.12", "PyQtWebEngine", "pygments"]`?

Reply:
Or I could leave this `"PyQtWebEngine"` out but instead document this additional requirement somewhere.

Reply:
I was thinking the first of these, unless there is a potential problem with that.

Reply:
You did make me become aware that PyQtWebEngine is only distributed on PyPI for PyQt5>=5.12. But did you mean that it is only _available_ for pyqt5>=5.12, rather than _needed_?

Specifying "pyqt5>=5.12" would to me means that there are reasons within Pyface that causes pyqt5<5.12 not to be supported, is that true?

So far the ">=5.12" bit seems to me a requirement of PyQtWebEngine, not a requirement of Pyface. If there is a distribution of PyQtWebEngine that would support PyQt5 5.11, I am not sure Pyface should disallow it (I don't think Pyface is setting these minimum requirements based on what are tested on CI - because if it does it would be very restrictive.)

I think only these two options currently make sense to me:
(1) `["pyqt5", "pygments"]`
(2) `["pyqt5", "PyQtWebEngine", "pygments"]`

In any case, these are extras requirements, so they are already optional - they may even be ignored by some installers (e.g. edm).

Perhaps I should revert the change to "extras_require" so we can focus on the changes in the import order which fixes the import error in #581, the error occurs even if one already has the required Qt web dependencies installed.